### PR TITLE
chore: split util/pod.go by function

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅰ. Describe what this PR does

Splitting util/pod.go by function is easy to maintain.

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/util/pod_resources_utils.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+package util
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+)
+
+// NOTE: functions in this file can be overwritten for extension
+
+func GetPodMilliCPULimit(pod *corev1.Pod) int64 {
```
Comment (saintube) [inline pkg/util/pod_resources_utils.go:26]: /lgtm

**Review by saintube — APPROVED** [review]

File: pkg/util/pod_qos_utils.go [hunk 2]
```diff
@@ -18,11 +18,12 @@ package util
 
 import (
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/apis/core/v1/helper/qos"
 
 	apiext "github.com/koordinator-sh/koordinator/apis/extension"
 )
 
-// NOTE: functions in this file can be overwrite for extension
+// NOTE: functions in this file can be overwritten for extension
```
Comment (jasonliu747) [inline pkg/util/pod_qos_utils.go:26]: 英语满分💯

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**